controllers/etcd/reconciler.go

```diff
@@ -77,7 +77,7 @@ type Reconciler struct {
 
 // NewReconciler creates a new reconciler for Etcd.
 func NewReconciler(mgr manager.Manager, config *Config) (*Reconciler, error) {
-	imageVector, err := utils.CreateImageVector(config.FeatureGates[string(features.UseEtcdWrapper)])
+	imageVector, err := utils.CreateImageVector()
```

Comment:
can we unify the import alias name and call it `ctrlutils` as we have done it in compaction reconciler?